File: omnipod-dash/src/main/java/info/nightscout/androidaps/plugins/pump/omnipod/dash/driver/pod/command/base/NonceEnabledCommand.java
package info.nightscout.androidaps.plugins.pump.omnipod.dash.driver.pod.command.base;

public abstract class NonceEnabledCommand extends HeaderEnabledCommand {
    protected final int nonce;

    protected NonceEnabledCommand(CommandType commandType, int uniqueId, short sequenceNumber, boolean multiCommandFlag, int nonce) {
        super(commandType, uniqueId, sequenceNumber, multiCommandFlag);
        this.nonce = nonce;
    }

}

File: pump/omnipod/dash/src/main/kotlin/app/aaps/pump/omnipod/dash/driver/pod/command/base/NonceEnabledCommand.kt
package app.aaps.pump.omnipod.dash.driver.pod.command.base

abstract class NonceEnabledCommand protected constructor(
    commandType: CommandType,
    uniqueId: Int,
    sequenceNumber: Short,
    multiCommandFlag: Boolean,
    protected val nonce: Int
) : HeaderEnabledCommand(commandType, uniqueId, sequenceNumber, multiCommandFlag)
